src/kvstore/NebulaStore.cpp

```diff
@@ -427,6 +427,24 @@ void NebulaStore::removeSpace(GraphSpaceID spaceId, bool isListener) {
   }
 }
 
+nebula::cpp2::ErrorCode NebulaStore::clearSpace(GraphSpaceID spaceId) {
+  folly::RWSpinLock::WriteHolder wh(&lock_);
+  auto spaceIt = this->spaces_.find(spaceId);
+  if (spaceIt != this->spaces_.end()) {
+    for (auto& part : spaceIt->second->parts_) {
+      auto ret = part.second->cleanup();
```

Comment:
Perhaps you need grab the raft lock here. BTW, do we need to block write by meta?